stock_picking_return_refund_option/hooks.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, SUPERUSER_ID, tools
+
+
+def _install_module_hook(cr):
+    if not tools.config.options.get('without_demo', False):
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        env['ir.module.module'].search([
```

Comment:
Why do you install manually these addons? If these addons must be installed, these should be declared into the manifest under the `depends`key. So it's clear for the user by reading the manifest that this module will install others modules.

Reply:
This is a trick for not depending on both modules and not creating 2 glue modules, but testing everything.

Reply:
You are wrong. The final result is that these modules are isntalled.

Reply:
Only in demo mode, not in production

Reply:
I strongly disagree with this kind of practice. Tests are  always run in a wrong context.

Reply:
Why a wrong context?

Reply:
Either the module depends on these two other modules or it does not depend on them. If it does not depend on it, we must be able to test them without them being installed. With this trick it will never be possible to test this module without both dependencies and thus ensure that it can work without them.

Reply:
There's no code to test are not installed, so it's not a problem. It's only a new field, that we put together for not needing a base module with the field plus 2 extra modules with the logic.